### PR TITLE
Add Image WorldObject

### LINKED ISSUE #225: An Image WorldObject
Like Volume. We can now create a plane with a texture, but that does not position the data as you'd normally want. Plus it makes sense from the point of colormapping: In a Plane with a tex, that text is the map. In an Image with a tex, that tex is the colormap (of any dimension). 

Relates to #104

### PR DESCRIPTION
**Pull request by almarklein**
Closes #225.

Looks like a lot of code, but most is copied from the volume and adjusted for 2D.

### CODE REVIEW

File: pygfx/objects/_more.py [hunk 1]
```diff
@@ -59,10 +59,37 @@ class Mesh(WorldObject):
     """
 
 
+class Image(WorldObject):
+    """An object  representing a 2D image in space.
```
Comment (Korijn) [inline pygfx/objects/_more.py:63]:
```suggestion
    """An object representing a 2D image in space.
```

File: pygfx/renderers/wgpu/volumerender.py [hunk 2]
```diff
@@ -462,7 +462,7 @@ def vertex_shader(self):
 
             // Store values for fragment shader
             varyings.position = vec4<f32>(ndc_pos);
-            varyings.world_pos = vec3<f32>(world_pos);
+            varyings.world_pos = vec3<f32>(world_pos.xzy);
```
Comment (Korijn) [inline pygfx/renderers/wgpu/volumerender.py:465]: you sure about this? remains unexplained in the PR description

Comment (almarklein) [reply pygfx/renderers/wgpu/volumerender.py:465]: world_pos is `vec4` but is cast to `vec3` here. I was not even aware that it was allowed to do that this way. Adding the `.xyz` makes it more explicit.

Comment (ivoflipse) [reply pygfx/renderers/wgpu/volumerender.py:465]: Except that the code says xzy instead of xyz

Comment (almarklein) [reply pygfx/renderers/wgpu/volumerender.py:465]: Oooh, shit.

Comment (almarklein) [reply pygfx/renderers/wgpu/volumerender.py:465]: Good catch. My bad!

Comment (almarklein) [inline pygfx/renderers/wgpu/volumerender.py:465]:
```suggestion
            varyings.world_pos = vec3<f32>(world_pos.xyz);
```

**Comment by Korijn** [comment]
LGTM!

**Pull request merged**